addons/mail/static/src/core/common/thread_model.js

```diff
@@ -541,6 +547,14 @@ export class Thread extends Record {
         return this.selfMember?.localMessageUnreadCounter > 0;
     }
 
+    get rpcParams() {
+        return {
+            ...(this.token ? { token: this.token } : {}),
+            ...(this.hash ? { hash: this.hash } : {}),
+            ...(this.pid ? { pid: this.pid } : {}),
+        };
```

Comment:
Shouldn't these be defined in portal only?
Also the actual field name of the portal mixin is `access_token` we should keep the same name for the JS field.